complex/ft_comp.c

```diff
@@ -54,8 +54,6 @@ static int ft_open_cntrs(void)
 {
 	int ret;
 
-	opts.comp_method = test_info.cq_wait_obj;
-
```

Comment:
I see your issue. I was considering the cq_wait_obj as the completion method, rather than it being cq-specific.
Then I think the thing to do would be to change "cq_wait_obj" to "comp_method" to generalize to CQs and counters using the FT_... enums.